packages/sdk/src/chainConfig/redemptionStrategies.ts

```diff
@@ -1,71 +1,118 @@
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.WBNB),
+    ],
+    [underlying(bscAssets, assetSymbols["BUSD-USDT"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.BUSD),
+    ],
+    [underlying(bscAssets, assetSymbols["BUSD-BTCB"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.BUSD),
+    ],
+    [underlying(bscAssets, assetSymbols["USDC-BUSD"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.BUSD),
+    ],
+    [underlying(bscAssets, assetSymbols["USDC-ETH"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.USDC),
+    ],
+    [underlying(bscAssets, assetSymbols["BTCB-BOMB"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.BTCB),
+    ],
+    [underlying(bscAssets, assetSymbols["BTCB-ETH"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.BTCB),
+    ],
+    [underlying(bscAssets, assetSymbols["CAKE-WBNB"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.WBNB),
+    ],
   },
   [SupportedChains.moonbase_alpha]: {},
   [SupportedChains.chapel]: {
-    [bscAssets.find((a: SupportedAsset) => a.symbol === assetSymbols["WBNB-BUSD"])!.underlying]:
-      RedemptionStrategy.UniswapLpTokenLiquidator,
+    [underlying(bscAssets, assetSymbols["WBNB-BUSD"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(bscAssets, assetSymbols.WBNB),
+    ],
   },
   [SupportedChains.aurora]: {},
   [SupportedChains.moonbeam]: {
-    [moonbeamAssets.find((a: SupportedAsset) => a.symbol === assetSymbols["GLMR-USDC"])!.underlying]:
-      RedemptionStrategy.UniswapLpTokenLiquidator,
-    [moonbeamAssets.find((a: SupportedAsset) => a.symbol === assetSymbols["GLMR-GLINT"])!.underlying]:
-      RedemptionStrategy.UniswapLpTokenLiquidator,
+    [underlying(moonbeamAssets, assetSymbols["GLMR-USDC"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(moonbeamAssets, assetSymbols.WGLMR),
+    ],
+    [underlying(moonbeamAssets, assetSymbols["GLMR-GLINT"])]: [
+      RedemptionStrategyContract.UniswapLpTokenLiquidator,
+      underlying(moonbeamAssets, assetSymbols.WGLMR),
+    ],
   },
   [SupportedChains.evmos_testnet]: {},
   [SupportedChains.evmos]: {
-    "0x -- Kinesis": RedemptionStrategy.CurveLpTokenLiquidatorNoRegistry,
+    "0x -- Kinesis": [RedemptionStrategyContract.CurveLpTokenLiquidatorNoRegistry, "ignored"],
   },
   [SupportedChains.ganache]: {},
   [SupportedChains.neon_devnet]: {},
   [SupportedChains.polygon]: {},
 };
 
+function underlying(assets: SupportedAsset[], symbol: string): string {
```

Comment:
nice. maybe we place it somewhere else, possibly in the `chainDeploy/helpers/utils.ts` ?